core/src/main/java/hudson/model/DirectoryBrowserSupport.java

```diff
@@ -404,12 +404,18 @@ private static void zip(OutputStream outputStream, VirtualFile dir, String glob)
          */
         private final boolean isReadable;
 
-        public Path(String href, String title, boolean isFolder, long size, boolean isReadable) {
+       /**
+        * For a file, the last modified timestamp.
+        */
+        private final long lastModified;
+
+        public Path(String href, String title, boolean isFolder, long size, boolean isReadable, long lastModified) {
```

Comment:
Changes signature of un-`@Restricted` constructor.

Reply:
Yes, this is a 🐛

Reply:
The old constructor was unrestricted, I will restrict it if you want.